tests/doc/run_doc_python_server_api_examples.py

```diff
@@ -2,6 +2,7 @@
 from time import sleep
```

Comment:
I also commented on this file, there are tickets for all of it already I think:

https://github.com/bigchaindb/bigchaindb/pull/590#issuecomment-249197867